autoload/tagbar.vim

```diff
@@ -534,6 +567,20 @@ endfunction
 
 " s:CheckFTCtags() {{{2
 function! s:CheckFTCtags(bin, ftype) abort
+    " dart_ctags isn't installed as a bin
+    " must use pub command to run
```

Comment:
I don't understand this comment. On which platform would it not be installed as a binary? And what exactly _is_ it? Excuse my ignorance of whatever tech is being employed here

Reply:
`pub` is dart's package manager, but it only installs the cli programs to the `pub` path.  so, you have to invoke `pub global run dart_ctags:tags` to run it.  it's possible to clone the repo and compile it into a binary and include that in your `PATH`, but i was considering maximum compatibility/ease of use.

Reply:
> so, you have to invoke pub global run dart_ctags:tags to run it

Okay, in that case this PR doesn't seem like it will quite cover things as it just passes the unmodified `a:bin` back as the thing to be executed. You might be using `pub` to _find_ if the specialty provider is installed but I don't see how would actually get executed as that is not part of the path & command that actually get run.

Reply:
oops I forgot to add the return if it fails.  I figured we don't want to run to generate tags on a random directory, so that's why I did the pub global list command to check if it was installed.